util.rb

```diff
@@ -0,0 +1,54 @@
+require 'sinatra/base'
+require_relative 'environment'
+
+# IAM - a resource usage metric collection and reporting system
+class Util
+  def self.val_id(data)
+    # checks if the hashes are empty
+    # checks if the values of id are fixed numbers
+    return 0 if data.empty?
+    data.each do |item|
+      id = item[:id]
+      return false if id.class != Fixnum
+    end
+    true
```

Comment:
Is this line necessary?  What happens if you remove the `true` in this method and the method that follows?  Do the methods return the same results?  Which one is more idiomatic?